### PR TITLE
Feature/behov for flere filtreringer p en gang

### PR DESCRIPTION
**Pull request by mimse**
## References

- #0000

### CODE REVIEW

File: apps/dh/api-dh/source/DataHub.WebApi/GraphQL/Types/Balance/BalanceResponsibilityMeteringPointType.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+namespace Energinet.DataHub.WebApi.GraphQL.Types.Balance;
+
+public class BalanceResponsibilityMeteringPointType : EnumType<ESett.MeteringPointType>
+{
+    protected override void Configure(IEnumTypeDescriptor<ESett.MeteringPointType> descriptor)
+    {
+        descriptor.Name("BalanceResponsibilityMeteringPointType");
+        descriptor.AsIsCase();
+    }
+}
```
Comment (dzhavat) [inline apps/dh/api-dh/source/DataHub.WebApi/GraphQL/Types/Balance/BalanceResponsibilityMeteringPointType.cs:27]: issue: looks like this causes an issue with the metering point type translations in the overview table. The value is not translated anymore.

image

File: libs/dh/shared/data-access-mocks/src/lib/data/esett-balance-responsible-messages.ts [hunk 2]
```diff
@@ -39,13 +39,13 @@ export const eSettBalanceResponsibleMessages = (apibase: string): BalanceRespons
       receivedDateTime: new Date('2021-02-01T00:00:00.000Z'),
       supplier: '123',
       balanceResponsible: '321',
-      gridArea: '344',
+      gridAreas: '344',
```
Comment (dzhavat) [inline libs/dh/shared/data-access-mocks/src/lib/data/esett-balance-responsible-messages.ts:42]: nitpick: this should probably stay `gridArea`

```suggestion
      gridArea: '344',
```

File: libs/dh/shared/data-access-mocks/src/lib/data/esett-balance-responsible-messages.ts [hunk 3]
```diff
@@ -89,7 +89,7 @@ export const eSettBalanceResponsibleMessages = (apibase: string): BalanceRespons
       balanceResponsible: '999',
       gridArea: '000',
       gridAreaWithName: null,
-      meteringPointType: TimeSeriesType.Production,
+      meteringPointType: BalanceResponsibilityMeteringPointType.Production,
```
Comment (dzhavat) [inline libs/dh/shared/data-access-mocks/src/lib/data/esett-balance-responsible-messages.ts:92]: suggestion: let's remove the type casting on line 98 in order to get better type checking.

File: apps/dh/api-dh/source/DataHub.WebApi/GraphQL/Types/Balance/BalanceResponsibilityMeteringPointType.cs (hunk 1 above)
Comment (mimse) [reply apps/dh/api-dh/source/DataHub.WebApi/GraphQL/Types/Balance/BalanceResponsibilityMeteringPointType.cs:27]: Same translation was used for both timeseriestype and metering point time. Fixed now

**Pull request merged**